src/graph/create-space.ts

```diff
@@ -0,0 +1,29 @@
+import { DEFAULT_API_HOST } from './constants.js';
+
+type Params = {
+  editorAddress: string;
+  name: string;
+  options?: {
+    apiHost?: string;
```

Comment:
We probably don't want to expose an API host yet due to our relying on specific IPFS nodes for writing/reading to IPFS. If we let people pass their own hosts with their own nodes they could write data that doesn't get picked up by the indexer.

Eventually we'll decentralize this so people can write to any IPFS node.

Reply:
removed and pushed